tools/scripts/tidy.toml

```diff
@@ -98,4 +98,46 @@ if greater_than ${output_length} 0
         trigger_error ${msg}
     end
 end
-'''
\ No newline at end of file
+'''
+
+[tasks.copy-license]
+description = "Copy the license file to each component for packaging."
+category = "ICU4X Development"
+script_runner = "@duckscript"
+script = '''
+exit_on_error true
+
+glob_pattern = set "./**/Cargo.toml"
+license = canonicalize LICENSE
+
+handle = glob_array ${glob_pattern}
+for path in ${handle}
+    root_dir = dirname ${path}
+    if not is_empty ${root_dir}
+        echo "Copying LICENSE to ${root_dir}/LICENSE"
+        cp ${license} ${root_dir}/LICENSE
+    end
+end
+'''
+
+[tasks.license-file-check]
+description = "Check that each component has a LICENSE file."
+category = "ICU4X Development"
+script_runner = "@duckscript"
+script = '''
+exit_on_error true
+
+glob_pattern = set "./**/Cargo.toml"
+license = canonicalize LICENSE
+
+handle = glob_array ${glob_pattern}
+for path in ${handle}
+    root_dir = dirname ${path}
+    if not is_empty ${root_dir}
+        if not is_path_exists ${root_dir}/LICENSE
```

Comment:
suggestion (nb): should we check that it is the same license, too?

Reply:
something like `if cmp -s ${root_dir}/LICENSE ${root_dir}/../LICENSE` (don't know my duckscript)

Reply:
I tried to do this prior to pushing this up for the initial review, but I was unable to get it working in duckscript:
```
license_text = readfile ${license}

if not eq ${license_text} ${license_text}
    echo "not working"
end
```
but it turns out that this works:
```
license_text = readfile ${license}

license_equal = eq ${license_text} ${license_text}
if not eq ${license_equal}
    echo "not working"
end
```
Duckscript is not intuitive to me at all.